### ISSUE TITLE
Error messages not complete in 'save as' window.

### ISSUE THREAD

**Issue by luciansmith** [protegeproject/protege#1243]
We have an ontology we usually work with in OWL format, but sometimes export to OBO.  OBO is more restrictive, so errors can creep in.  When you load the OWL file and then 'Save As' and select the OBO format, it will fail, but the error message is cut off and it's impossible to know what the problem actually is:

![image](https://github.com/user-attachments/assets/4db8fd2f-4871-424d-bb62-841a5b501454)

(in this case, it was an ObjectProperty with two 'domain' children.  But we couldn't tell *which* ObjectProperty, nor even that it *was* an ObjectProperty.)

**Comment by ykazakov** [comment]
Can you see this error in the Protege log view?

**Comment by luciansmith** [comment]
I don't know what the log view is.  I didn't find anything promising in the menus when I perused it--how do I get to it?

**Comment by gouttegd** [comment]
It’s under the _Window_ menu, under the name _Show log_.

It is also accessible directly through the small button at the bottom right corner of the main window, just next to to the “Show inferences” checkbox:

![protege-log-window-button](https://github.com/user-attachments/assets/487f8c4b-82ae-4150-8745-3137b90bf9d7)

**Comment by ykazakov** [comment]
Any error in Protege should trigger a red exclamation mark in the bottom right corner, wich should be difficult to miss:

<img width="877" alt="Screenshot 2024-11-28 at 10 45 30" src="https://github.com/user-attachments/assets/8bf477ff-62ae-43a1-b25a-5d03879d1796">

Clicking on this icon should bring up the log (and clear the error status).

One possible way one could improve upon that is to add a button to an error dialog window that would also open the log. (Clearly, we cannot cram all error messages into the dialog window).

**Comment by ykazakov** [comment]
> having word wrap on by default would have been nice

The word wrap comes with some performance penalty (if one resizes the log window) that's why, I think, it is disabled by default.

**Comment by luciansmith** [comment]
Nope, totally missed the red triangle.  It's a bit smaller (relatively) on my own screen:
![image](https://github.com/user-attachments/assets/d14f57f2-dcd9-4b2d-b0a8-54a8a3a9d503)
But, it's good to know it's there!

I think the problem is that the window promises that it is telling you what the error message is, and then it does not.  So, one assumes that the error message is inaccessible.  I was told to use robot to convert my ontology because there, the error message would be present and nicely formatted, and that Protege didn't have the error message at all.

Perhaps the cut-off error message could be clicked, and would take you to the log?
